lib/gakubuchi/template.rb

```diff
@@ -68,7 +68,8 @@ def assets
 
     def extract_extname(path)
       extname = path.extname
-      extname.empty? ? extname : "#{extract_extname(path.basename(extname))}#{extname}"
+      extname.empty? || extname == ".html" ?
+        extname : "#{extract_extname(path.basename(extname))}#{extname}"
```

Comment:
@hilohiro 
Although `extract_extname` returns `.html.erb` as the extension of `foo.bar.baz.html`,
Do you think that is appropriate as the behavior of the method?

```
irb(main):001:0> Gakubuchi::Template.new(Rails.root.join("app/assets/templates/foo.bar.baz.html")).extname
=> ".html"
```